src/log-receiver/log-receiver.service.spec.ts

```diff
@@ -29,6 +36,19 @@ describe('LogReceiverService', () => {
     expect(service).toBeDefined();
   });
 
+  // it('should handle log', () => {
+  //   const log: LogMessageFormat = {
+  //     time: new Date().getTime(),
+  //     type: LogType.CPU,
+  //     detectorUrl: "http://localhost:3000/",
+  //     sourceUrl: null,
+  //     data: {
+  //       cpuUtilization: 100
+  //     }
+  //   }
+  //   expect(service.handleLogMessage(log)).toBeDefined(); // Can't report Issues in test...
+  // })
+
```

Comment:
wenn ihr den test nicht nutzt, mach ihn ganz weg. auskommentierter code ist doof. den  vergisst man gerne mal und dann schimmelt er auf ewig in der datei rum und keiner weiß mehr warum.